reactor-core/build.gradle

```diff
@@ -212,29 +212,15 @@ task kdocZip(type: Zip, dependsOn: dokka) {
   from("${project.buildDir}/docs/kdoc")
 }
 
-task testStaticInit(type: Test, group: 'verification') {
-  systemProperty 'reactor.trace.operatorStacktrace', 'true'
-  include '**/*TestStaticInit.*'
-  doFirst {
-	println "Additional tests from `testStaticInit` ($includes)"
-  }
-  afterSuite { TestDescriptor descriptor, TestResult result ->
-	if (result.testCount > 0 && (result.skippedTestCount + result.failedTestCount + result.successfulTestCount == 0)) {
-	  throw new GradleException("No static initialization tests were executed")
-	}
-  }
-}
 
 task loops(type: Test, group: 'verification') {
-  mustRunAfter testStaticInit
```

Comment:
@simonbasle these tests were triggered by a chain (`check` -> `testStaticInit` -must-run-after-> `loops`) but now they are not executed at all.
Consider adding `dependsOn(loops)` to `check`'s config

Reply:
it's there, conditional on the version being a `.RELEASE` (loops are just calling normal single-run tests in big loops btw). the `mustRunAfter` was there so that static init test was guaranteed to be ordered first, then the other tests would run (so that they don't reset the hook by running before the staticInitTest) 👍

Reply:
see https://github.com/reactor/reactor-core/blob/3.2.x/reactor-core/build.gradle#L247